src/pe/utils.rs

```diff
@@ -22,6 +22,23 @@ fn aligned_pointer_to_raw_data(pointer_to_raw_data: usize) -> usize {
     pointer_to_raw_data & !PHYSICAL_ALIGN
 }
 
+// Performs arbitrary alignment of values
+// based on homogeneous numerical types.
+#[inline]
+pub fn align_to<N>(value: N, align: N) -> N
```

Comment:
This could be a more global utility, I can send a PR later to clean up manual alignments if needed.

Reply:
@m4b basically, this is a useful utility for anyone doing alignments, I can duplicate it downstream but it feels a bit weird as anyone doing any PE shit will have to deal with various alignments computations I'd say?

Reply:
Resolving, let me know if I should do something else.